source/_integrations/rfxtrx.markdown

```diff
@@ -96,6 +99,10 @@ Also, several switches and other devices will also expose sensor entities with b
 
 The RFXtrx integration support binary sensors that communicate in the frequency range of 433.92 MHz. The RFXtrx binary sensor integration provides support for them. Many cheap sensors available on the web today are based on a particular RF chip called *PT-2262*. Depending on the running firmware on the RFXcom box, some of them may be recognized under the X10 protocol, but most of them are recognized under the *Lighting4* protocol. The RFXtrx binary sensor integration provides some special options for them, while other RFXtrx protocols should work too.
 
+#### Sirens
+
+The RFXtrx integration supports sirens entites for a few types of security systems and chimes. This entity allow triggering the chime or siren from Home Assistant as well as monitoring their status. Most of chimes and security systems need a configured off-delay to work correctly since they only transmit when active.
```

Comment:
```suggestion
The RFXtrx integration supports siren entites for a few types of security systems and chimes. This entity allows triggering the chime or siren from Home Assistant as well as monitoring their status. Most of the chimes and security systems need a configured off-delay to work correctly since they only transmit when active.
```

Reply:
entites -> entities